Scoop/View/EditProfileViewController.swift

```diff
@@ -0,0 +1,256 @@
+            make.leading.trailing.equalToSuperview().inset(leadingTrailingInset)
+            make.top.equalTo(profileImageView.snp.bottom).offset(25)
+        }
+        
+        nameTextField.textColor = UIColor.black
+        nameTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(nameTextField)
+        
+        nameTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        pronounsTextField.textColor = UIColor.black
+        pronounsTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(pronounsTextField)
+        
+        pronounsTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        hometownTextField.textColor = UIColor.black
+        hometownTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(hometownTextField)
+        
+        hometownTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+        
+        classTextField.textColor = UIColor.black
+        classTextField.borderStyle = .roundedRect
+        stackView.addArrangedSubview(classTextField)
+        
+        classTextField.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.height.equalTo(textFieldHeight)
+        }
+    }
+    
+    // MARK: - Helper Functions
+    
+    private func updateUser() {
+        let name = nameTextField.text?.split(separator: " ")
+        let firstName = String(name?[0] ?? "")
+        let lastName = String(name?[1...].joined(separator: " ") ?? "")
+        let grade = classTextField.text
+        let pronouns = pronounsTextField.text
+        
+        hometown = hometownTextField.text
+        
+        var userAnswers: [UserAnswer] = []
+                
+        user?.prompts.forEach { prompt in
+            if prompt.questionName == "Hometown" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: hometown ?? ""))
+            } else if prompt.questionName == "Talkative" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: String(talkative ?? 0)))
+            } else if prompt.questionName == "Music" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: String(music ?? 0)))
+            } else if prompt.questionName == "Snack" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: snack ?? ""))
+            } else if prompt.questionName == "Song" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: song ?? ""))
+            } else if prompt.questionName == "Stop" {
+                userAnswers.append(UserAnswer(id: prompt.id, answer: stop ?? ""))
+            }
```

Comment:
I would use a switch statement here to make the code more robust. You will need to change the type to an enum. This may be difficult if the rest of the codebase uses a String for prompt.questionName but this variable should be stored as an enum ideally. Decoding will still work with an enum.